frontend/e2e/annotation-mode-page/annotation-mode.page.spec.ts

```diff
@@ -10,19 +10,40 @@ import {
   mockRecordForLongAndShortQuestion,
 } from "../common";
 
+const shortcuts = Object.freeze({
```

Comment:
Why do you use an Object freeze ?

Reply:
Only to prevent the modification outside of the object, but I can remove if not necessary